src/redturtle/prenotazioni/actions/mail.py

```diff
@@ -64,10 +69,13 @@ def get_recipients(self):
         return list(filter(bool, recipients))
 
     def manage_attachments(self, msg):
+        booking = self.event.object
```

Comment:
anche qui, siamo sempre sicuri che in self.event.object ci sia un booking? non va usato get_target_object e poi controllato che sia una prenotazione ("booking")?

Reply:
nì..perché in target_obj c'è il content-type che contiene il campo con l'email.
Se si imposta "parent", allora è diverso da self.event.object (che è il ct che scatena l'evento)

Reply:
ok vedo comunque che poi il controllo che sia prenotazione lo fai.